airflow/providers/google/cloud/operators/dataflow.py

```diff
@@ -612,6 +613,7 @@ def __init__(
         environment: Optional[Dict] = None,
         cancel_timeout: Optional[int] = 10 * 60,
         wait_until_finished: Optional[bool] = None,
+        append_job_name: Optional[bool] = True,
```

Comment:
Why optional? None is not acceptable in DataflowHook.

Reply:
Removed optional, thanks